Rename Runtime Manager + update related terms (#6071)
Kong/docs.konghq.com: app/konnect/gateway-manager/index.md

Access to each control plane is configurable on a team-by-team basis using
entity-specific permissions. For more information, see [Administer teams](/konnect/org-management/teams-and-roles/).

### {{site.base_gateway}} control planes

Every region in every organization starts with one default control plane.
This control plane can't be deleted, and its status as the default can't be changed.

With an [Enterprise subscription](https://konghq.com/pricing/), you can configure additional
custom control panes. Use multiple control planes in one {{site.konnect_short_name}} organization to
manage data plane nodes and their configuration in any groupings you want.

Some common use cases for using multiple control planes include:

* **Environment separation:** Split environments based on their purpose, such as
development, staging, and production.
* **Region separation:** Dedicate each control plane to a region or group of
regions. Spin up data plane nodes in those regions for each control plane.
* **Team separation:** Dedicate each control plane to a different team and share
resources based on team purpose.

![control planes](/assets/images/docs/konnect/konnect-runtime-groups-example.png)

> > _**Figure 1:** Example control plane configuration with three control planes: the default, a development plane, and a production plane. {{site.konnect_short_name}} is the SaaS-managed global management plane that manages all of the control planes, while the control planes manage configuration for data plane nodes._

Reviewer: ```suggestion
> _**Figure 1:** Example control plane group configuration for three control planes: the default, a development CP, and a production CP. {{site.konnect_short_name}} is the SaaS-managed global management plane that manages all of the control planes, while the control planes manage configuration for data plane nodes._
```

Reviewer: it feels odd to just use the word plane.

for different purposes.
* **Workspaces in the cloud**: Control plane groups function similarly to {{site.base_gateway}} workspaces, with the added benefit of a cloud control plane.

Learn more about control plane groups:
* [Intro to control plane groups](/konnect/gateway-manager/control-plane-groups/)
* [Set up and manage control plane groups](/konnect/gateway-manager/control-plane-groups/how-to/)
* [Migrate configuration into a control plane group](/konnect/gateway-manager/control-plane-groups/migrate/)
* [Conflicts in control plane groups](/konnect/gateway-manager/control-plane-groups/conflicts/)

### Control plane dashboard

For each control plane, you can view traffic, error rate, and {{site.base_gateway}} service analytics for its data plane nodes. 
This allows you to see how much of a control plane is used. You can also select the time frame of analytics that you want to display.

### Deleting a control plane

{:.warning}
> **Warning:** Deleting a control plane is irreversible. Make sure that you are
certain that you want to delete the control plane, and that all entities and data plane

> nodes in control plane the have been accounted for.

Reviewer: ```suggestion
nodes in the control plane the have been accounted for.
```

When a control plane is deleted, all associated entities are also deleted.
This includes all entities configured in the Gateway Manager for this control plane.
As a best practice, [back up](/konnect/gateway-manager/backup-restore/) a 
control plane's configuration before deleting it to avoid losing necessary configuration.

Data plane nodes that are still active when the control plane is deleted will not be
terminated, but they will be orphaned. They will continue processing traffic
using the last configuration they received until they are either connected to
a new control plane or manually shut down.

You cannot delete the default control plane.

## Data plane nodes

A data plane node is a single {{site.base_gateway}} instance. 
Data plane nodes service traffic for the control plane. 

Kong does not host {{site.base_gateway}} data plane nodes. 
You must deploy your own nodes, either on your own systems or in 
an external cloud provider.

The Gateway Manager simplifies data plane node deployment 
by providing a script to provision a {{site.base_gateway}} data plane node in a 

> Docker container running Linux, on MacOS, or on Windows, eliminating confusion about
> initial configuration or setup.

Reviewer: ```suggestion
Docker container running Linux, on MacOS, or on Windows. 
```
